scalameta/tokenizers/native/src/main/scala/scala/meta/internal/tokenizers/PlatformTokenizerCache.scala

```diff
@@ -0,0 +1,20 @@
+package scala.meta
+package internal
+package tokenizers
+
+import scala.collection.mutable
+import scala.meta.inputs._
+import scala.meta.tokens._
+
+import java.{util => ju}
+
+object PlatformTokenizerCache {
```

Comment:
Ditto, we should not copy-paste between js/native since that will be hard to maintain